### DISCUSSION TITLE
Pursh Resume Review

### DISCUSSION THREAD

**Discussion by sanjay-kv** [recodehive/resume-pitch#106] [Q&A] [answered]
[pursh.pdf](https://github.com/user-attachments/files/17109582/pursh.pdf)
[ATS Version of CV (30).docx](https://github.com/user-attachments/files/17109583/ATS.Version.of.CV.30.docx)

**Comment by kavyapurushottama** [comment, marked as answer]
Thank you sir (@sanjay-kv) for reviewing my resume and giving me your valuable suggestions.
